src/main/java/com/kllhy/roadmap/roadmap/domain/model/RoadMap.java

```diff
@@ -0,0 +1,73 @@
+package com.kllhy.roadmap.roadmap.domain.model;
+
+import com.kllhy.roadmap.common.model.AggregateRoot;
+import com.kllhy.roadmap.roadmap.domain.model.creation_spec.CreationRoadMap;
+import jakarta.persistence.*;
+import java.sql.Timestamp;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "road_map")
+public class RoadMap extends AggregateRoot {
+
+    @Column(name = "title", nullable = false)
+    private String title;
+
+    @Column(name = "description")
+    private String description;
+
+    @Column(name = "deleted_at")
+    private Timestamp deletedAt;
+
+    @Column(name = "is_deleted", nullable = false)
+    private boolean isDeleted;
+
+    @Column(name = "is_draft", nullable = false)
+    private boolean isDraft;
+
+    @Column(name = "category_id", nullable = false)
+    private Long category_id;
```

Comment:
```suggestion
    private Long categoryId;
```
카멜 케이스로 통일해서 category_id 필드명을 `categoryId` 로 바꿨으면 합니다!

Reply:
넵 반영하겠습니다!